### PR TITLE
document docker config.json

### PR DESCRIPTION
**Pull request by ahobson**

### CODE REVIEW

**Comment by sojeri** [comment]
so many force pushes!

File: developing/docker/README.md [hunk 1]
```diff
@@ -87,3 +87,26 @@ unused networks:
...
+      "detachKeys": "ctrl-@,ctrl-["
+    }
+
+### Credentials Store
+
+You can store docker credentials in an [external credentials
+store](https://docs.docker.com/engine/reference/commandline/login/#credentials-store).
+
+On macOS, the keychain will be used by default, but if you need/want
+to store shared credentials, [pass](https://www.passwordstore.org/)
```
Comment (sojeri) [inline developing/docker/README.md]: is this a recommended tool over aws vault, 1password, or other tools someone might be using to store secrets?

if so, it should probably be documented at a higher level, not necessarily in the docker page here.

**Comment by sojeri** [comment]
what story is this related to?

**Comment by ahobson** [comment]
> what story is this related to?

It's not related to a story.  I remembered some helpful docker hints and after I shared them in slack thought I should document them more permanently.

File: developing/docker/README.md (hunk 1 above)
Comment (ahobson) [reply developing/docker/README.md]: No recommendation intended, just pointing out capabilities of docker that could prove useful depending on the problem you are trying to solve.

Comment (rpdelaney) [reply developing/docker/README.md]: 1password has a commandline utility that can be installed with `brew install 1password-cli`. Can we suggest that instead, to keep everyone on the same tool?

Comment (ahobson) [reply developing/docker/README.md]: Sure!  Please update the PR to document how that works with docker.  I'm just passing along information already included in the docker documentation at the link I provided.

**Pull request merged**